api/test/integration/env/configurations/rbac/rule/black_config.yaml

```diff
@@ -3,6 +3,5 @@
   - rules:read
   resources:
   - rule:file:0010-rules_config.xml
-  - rule:file:0015-ossec_rules.xml
```

Comment:
Are we sure about this removal? Shouldn't `0015-wazuh_rules.xml` go here?

Reply:
This change is part of [#6831](https://github.com/wazuh/wazuh/pull/6831) and was validated by the framework team.